### DISCUSSION TITLE
Parameter values overhaul from 0-50 to actual values

### DISCUSSION THREAD

**Discussion by o010b** [SynthstromAudible/DelugeFirmware#601] [Ideas]
Now with the OLED it would be nice to be able to have actual values and also a little higher range and precision. I'm thinking to move from the arbitrary -50 - 0 - +50 to actual dB, milliseconds etc. 

The file format should make it easy enough to detect if songs are stored with old values and use a lookup table to translate into new. 

C/C++ is beyond my skills, but I'd be glad to contribute with boring manual labor or calculating tables and such.

**Comment by ok-reza** [comment]
an interesting idea, but it would take a lot of iteration to make sure things look right in the UI. when you adjust a parameters value, like frequency, the number is displayed as a large bold number 0-50 (OLED). if anything is over 3 digits long then it will clip into the text of the parameter title. going from 3 digits (-50) to multiple (eg. 250hz, 4250khz, etc) would cause a lot of issues. the whole UI look would probably have to be reconsidered tbh.

i'm a fan of eventually being able to switch from 0-50 deluge values to 0-127 midi values, but that also creates issues of how bipolar values are handeled, ideally it would be -127 to 127 but in reality need to be -63 to 63 to be compatible with midi.
